lib/SymMessageParser/index.js

```diff
@@ -2,14 +2,13 @@ var SymBotAuth = require('../SymBotAuth')
 
 var SymMessageParser = {}
 
-const regex = /(<div.+?>)(.*)(<\/div>)/g
+const regex = /^(<div.+?>)(.*)(<\/div>)$/g
```

Comment:
could it be a problem if we have _whitespace_<div>.... ?

Reply:
Indeed... Let me update that.